grpc/service_config/service_config.proto

```diff
@@ -622,6 +642,8 @@ message LoadBalancingConfig {
       [json_name = "xds_wrr_locality_experimental"];
     LeastRequestLocalityLoadBalancingPolicyConfig least_request_experimental = 17
       [json_name = "least_request_experimental"];
+    OverrideHostLoadBalancingPolicyConfig override_host_experimental = 18
```

Comment:
Please move this up to line 632.  This policy is not deprecated, so it doesn't belong in this section (see comment on line 633).

(There are some existing entries here that also need to move up, which I'm fixing in #115.)